templates/miq-template.yaml

```diff
@@ -176,6 +241,10 @@ objects:
            name: "${NAME}-region"
            persistentVolumeClaim:
              claimName: ${NAME}-region
+         -
+           name: "${NAME}-assets"
```

Comment:
I think I would rather have that than another PV .. is image size that much of a concern?

Additionally I feel like assets shouldn't be this closely tied to the app runtime. Is there a way to ship them in the image rather than having to persist them? Maybe we can grab the app code, compile them then remove the app code in the docker file?